model-optimizer/extensions/back/ConvolutionNormalizer.py

```diff
@@ -24,18 +24,18 @@ def resolve_convolution_with_group(node: Node, group: int, ir_version: str):
     assert len(weights_shape) in [3, 4, 5]
     assert weights_shape[0] % group == 0
 
-    assert int64_array(node.output).ndim == 0
     if ir_version == 'V7':
         if weights_shape[0] == node.output:
             # weights are already is in [G*O I X Y] format
             return
-        new_shape = int64_array([node.output, -1, *weights_shape[2:]])
-
+        new_shape = shape_array([node.output, -1, *weights_shape[2:]])
     elif ir_version == 'V10':
+        # TODO rewrite this transformation to generate a shape-computing sub-graph. Ticket 62076
         I = input_shape[1]
-        new_shape = int64_array([group, node.output / group, I / group, *weights_shape[2:]])
-        assert np.prod(weights_shape) == np.prod(new_shape), \
-            'Initial weights shape {}, grouped weights shape {}'.format(weights_shape, new_shape)
+        new_shape = shape_array([group, node.output // group, I // group, *weights_shape[2:]])
+        assert is_fully_defined(weights_shape[2:]) and is_fully_defined(I) and \
+               np.prod(weights_shape) == np.prod(new_shape), 'Initial weights shape {}, grouped weights shape {}' \
+                                                             ''.format(weights_shape, new_shape)
         del node['group']
```

Comment:
on line 44 int64_array(new_shape) is done. What will be a result for int64_array(masked_array)?

Reply:
This transformation will fail. That is why I left a comment in line 33 to rewrite this transformation. This will be done in a separate PR.